### PR TITLE
add repository field in backbone-super

### PR DESCRIPTION
**Pull request by Amomo**
Hi @Piicksarn 

For #5194, I add repository info in the package.json of backbone-super.
repo: https://github.com/lukasolson/backbone-super

Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `12`, Star `215`, Fork `31`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

File: ajax/libs/backbone-super/package.json [hunk 1]
```diff
@@ -8,5 +8,9 @@
     "super"
   ],
   "filename": "backbone-super-min.js",
-  "license": "MIT"
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lukasolson/backbone-super"
```
Comment (Amomo) [inline ajax/libs/backbone-super/package.json]: test

**Comment by PeterDaveHello** [comment]
Should rebase with our latest master branch.

**Comment by Amomo** [comment]
@PeterDaveHello I have rebased it to latest master branch.
Could you take a look? Thank you! :-)

**Comment by PeterDaveHello** [comment]
Let's add auto-update config.

**Pull request merged**